Tools/GenerateWebsiteData.ps1

```diff
@@ -62,11 +67,43 @@ foreach ($cmdlet in $cmdlets) {
         "documentation" = $docText
     }
 
+    # Contains more information than above.
+    $cmdletDocObjFull = @{
+        "name" = $name
+        "synopsis" = ($docObj.Synopsis | Out-String).Trim()
```

Comment:
Since the `  | Out-String).Trim()` pattern is used so often, could you just create another method? Such as `convertToString()`?